fast64_internal/sm64/sm64_objects.py

```diff
@@ -1845,6 +1850,59 @@ def upgrade_changed_props():
             obj.fast64.sm64.version = SM64_ObjectProperties.cur_version
 
 
+class SM64_SegmentProperties(bpy.types.PropertyGroup):
+    group5_seg_str: bpy.props.StringProperty(name="Segment 5 Seg")
+    group5_str: bpy.props.StringProperty(name="Segment 5 Group")
+    group6_seg_str: bpy.props.StringProperty(name="Segment 6 Seg")
+    group6_str: bpy.props.StringProperty(name="Segment 6 Group")
+    group5_enum: bpy.props.EnumProperty(name="Segment 5 Group", default="Do Not Write", items=groupsSeg5)
+    group6_enum: bpy.props.EnumProperty(name="Segment 6 Group", default="Do Not Write", items=groupsSeg6)
+
+    def draw(self, layout):
+        col = layout.column()
+        prop_split(col, self, "group5_enum", "Segment 5 Select")
+        if self.group5_enum == "Custom":
+            prop_split(col, self, "group5_seg_str", "Segment 5 Seg")
+            prop_split(col, self, "group5_str", "Segment 5 Group")
+        col = layout.column()
+        prop_split(col, self, "group6_enum", "Segment 6 Select")
+        if self.group6_enum == "Custom":
+            prop_split(col, self, "group6_seg_str", "Segment 6 Seg")
+            prop_split(col, self, "group6_str", "Segment 6 Group")
+
+    def convert_group(self, grp):
+        num = int(re.findall("\d+", grp)[-1]) + 1
```

Comment:
```suggestion
        num = int(grp.removeprefix("group")) + 1
```
seems like a clearer intent to me
can "convert_group" have a better name? I can't really suggest anything due to not knowing what this is about but something like `group_to_script_func_name` idk :shrug:

Reply:
renamed the function to jump_link_from_enum.